app/templates/buyers/brief_overview.html

```diff
@@ -0,0 +1,139 @@
+{% extends "_base_page.html" %}
+
+{% import "macros/brief_links.html" as brief_links %}
+
+{% block page_title %}{{ brief.title or brief.lotName }} – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  <div class="grid-row">
+
+    {% block before_heading %}{% endblock %}
+    <div class="column-two-thirds">
+      {% with
+        heading = brief.get('title', brief['lotName']),
+        context = 'Overview'
+      %}
+        {% include "toolkit/page-heading.html" %}
+      {% endwith %}
+
+    {% block before_sections %}{% endblock %}
+
+      <ol class="instruction-list">
+
+        {% with
+          steps = [
+            {
+              'title': 'Write requirements',
+              'description': 'Before you can publish your requirements, you must complete:'
+            },
+            {
+              'title': 'Publish requirements',
+              'description': 'Before you can publish your requirements, you must complete:'
+            },
+            {
+              'title': 'Answer supplier questions',
+              'description': 'When you’ve published your requirements, you must answer all supplier questions.',
+              'links': [
+                {
+                  'href': '/',
+                  'text': 'How to manage supplier questions'
+                },
+                {
+                  'href': url_for(".clarification_questions",framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id),
+                  'text': 'Publish supplier questions and answers'
+                }
+              ]
+            },
+            {
+              'title': 'Choose the shortlist',
+              'description': 'After the application deadline, shortlist the suppliers who applied.',
+              'links': [
+                {
+                  'href': '/',
+                  'text': 'How to shortlist suppliers',
```

Comment:
Will be `https://www.gov.uk/guidance/how-to-shortlist-digital-outcomes-and-specialists-suppliers`